data/json/professions.json

```diff
@@ -4232,5 +4232,65 @@
     "points": -3,
     "items": { "both": [ "shorts", "heroin", "syringe" ], "male": [ "briefs" ], "female": [ "bra", "panties" ] },
     "addictions": [ { "intensity": 40, "type": "opiate" } ]
+  },
+  {
+    "type": "profession",
+    "ident": "yoga_instructor",
+    "name": "Yoga Instructor",
+    "description": "You were in the middle of downward dog. Now you need to get down and out.",
+    "points": 2,
+    "traits": [ "FAST_REFLEXES", "LIGHTEATER", "THINSKIN", "FLIMSY" ],
+    "skills": [ { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": {
+        "items": [ "runner_bag", "unitard", "lowtops", "socks_ankle", "wristwatch" ],
+        "entries": [ { "group": "charged_cell_phone" } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "sports_bra", "panties" ]
+    },
+    "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "runner",
+    "name": "Runner",
+    "description": "Forest Chump was your role model growing up. Time to make him proud.",
+    "points": 2,
+    "traits": [ "GOODCARDIO", "HUNGER", "FLEET", "FLIMSY" ],
+    "skills": [ { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": {
+        "items": [ "runner_bag", "b_shorts", "tank_top", "sneakers", "socks_ankle", "wristwatch" ],
+        "entries": [
+          { "group": "charged_cell_phone" },
+          { "group": "charged_mp3" },
+          { "item": "towel" },
+          { "item": "toothbrush_plain" },
+          { "item": "hairbrush" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "sports_bra", "panties" ]
+    },
+    "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "weight_lifter",
+    "name": "Lifter",
+    "description": "You were miring your gainz in the mirror when you saw the commotion outside.",
+    "points": 4,
+    "traits": [ "WEAKSTOMACH", "SMELLY", "CLUMSY", "STRONGBACK", "ADRENALINE", "GOODCARDIO", "ANTIJUNK" ],
```

Comment:
I can only see STRONGBACK and GOODCARDIO as justified.

Reply:
I agree weak stomach was a poor choice. I was supposed to choose the weakened immune system option. Excessive amounts of loading of muscle groups will leave you in with a supressed immune system.

Smelly was probably a bad idea.
Clumsy was a bad idea.

Adrenaline is a good choice. Explosiveness in lifting requires you to have a lot of carbohydrates before doing a prolonged workout. Glucose is released during an adrenaline rush. Glucose is also consumed during explosive lifting. Power lifters and strongmen will eat high carbohydrate meals before and during workouts. Strongmen are more likely to consume junk food since their diet primarily relies on calorie count after meeting their macros.

The lifting profession is a simplified version of all of the types of lifters. There really isnt a need to specifically create a profession for a bodybuilder, lifter, power lifter, strong man, cross fit, calisthenics, etc. They all do share quite a few comonalities.

Outside of being a strongman, the average lifter doesn't consume junkfood. Meeting your macros on a calorie restricted diet (cut) or even on a +500 calorie intake (bulk) would be jeopardized by eating junkfood. Outside of a few niche aspects of lifting (strongman, stone lifting, log throwing), junkfood is not a staple of the average lifter's diet.